srcpkgs/puddletag/template

```diff
@@ -1,13 +1,15 @@
 # Template file for 'puddletag'
 pkgname=puddletag
-version=2.0.1
-revision=3
+version=2.1.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3 python3-configobj python3-mutagen python3-parsing python3-PyQt5 python3-PyQt5-svg"
+depends="python3 python3-configobj python3-mutagen python3-parsing python3-PyQt5
+ python3-PyQt5-svg python3-pyacoustid"
+checkdepends="python3-wheel python3-pip"
```

Comment:
This just hides missing dependencies, fix it properly with something like:
```suggestion
checkdepends="python3-pytest $depends ..."
```

Reply:
thank you!

Reply:
hmm seems like it doesn't use pytest...

Reply:
also tried `checkdepends="python3-wheel python3-pip $depends"` but that didn't work either...

Reply:
Correct me if I'm wrong, but isn't it just because puddletag doesn't provide any test? So you can just add make_check=no (with comment that it doesn't have them)